quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -274,6 +276,8 @@ impl Default for SearcherConfig {
             split_cache: None,
             request_timeout_secs: Self::default_request_timeout_secs(),
             storage_timeout_policy: None,
+            warmup_memory_budget: ByteSize::gb(1),
```

Comment:
I think we want larger defaults here. 
Maybe memory budget of 10GB
and single split alloc to 1GB or something like this.